### ISSUE TITLE
Fix array flatten signature in docs

### ISSUE THREAD

**Issue by Tommytrg** [witnet/witnet-rust#539]
We should fix the definition of the flatten method [here](https://github.com/witnet/witnet-rust/blob/master/docs/protocol/data-requests/radon/types/array.md#arrayFlatten). `depth` arg is optional.

The correct syntax is:
```
  flatten(depth?: Int): Array<T>
```

**Comment by Sedherthe** [comment]
Hi!
I want to contribute to this issue! This is going to be my first contribution! So the fix for this issue is to make that one change in the syntax for the flatten function in docs in the given location ... Am I right!?

**Comment by Tommytrg** [comment]
@Sedherthe thanks for joining the Witnet community and starting your first contribution! 

Yes, you are right. By the other hand, don't forget to:
-  Verify your commit [with your PGP signature](https://help.github.com/en/articles/signing-commits).
- Write a commit message following the [conventional commits specification](https://www.conventionalcommits.org/en/v1.0.0-beta.2/). An example for this issue could be:
```
  docs(radon): fix array flatten signature
```

**Comment by Sedherthe** [comment]
Hi!
Thank you for the guidelines!!
I submitted the PR for this issue!!
Please let me know if there are any corrections.

**Comment by aesedepece** [comment]
This was closed by #589
